.github/workflows/ci.yaml

```diff
@@ -32,14 +32,15 @@ jobs:
       fail-fast: false
       matrix:
         # Test modern Odoo versions with latest Postgres version
-        pg_version: ["12"]
+        pg_version: ["13"]
         odoo_version:
```

Comment:
Shouldn't you add version 14.0 here? Or does that go on a different PR?

Reply:
Yes, that's in #327